soc/arm/ti_simplelink/cc13x2x7_cc26x2x7/Kconfig.soc

```diff
@@ -65,4 +65,11 @@ config CC13X2_CC26X2_BOOTLOADER_BACKDOOR_LEVEL
 	help
 	  Set the active level of the pin selected for the bootloader backdoor.
 
+config CC13X2_CC26X2_XOSC_CAPARRAY_DELTA
+	hex "Cap array tuning delta"
+	range 0 0xFF
+	default 0xD5
+	help
+	  Enable a specific cap array tunning delta.
+
```

Comment:
Changes looks same as https://github.com/zephyrproject-rtos/zephyr/pull/58877/commits/f0012322701cbec050447859c386b66b293bc49e , in such cases it would be good to retain the authorship of the original author and add your signed-off-by: